.github/workflows/main.yml

```diff
@@ -4,9 +4,9 @@ on:
   push:
     branches:
       - master
-    paths-ignore:
-      - 'docs/**'
-      - '*.md'
+    paths-include:
+      - "Dockerfile"
```

Comment:
This seems harder to maintain if files are added to the repo, people won't think to update this list

Reply:
There's no reason to expand this image any further IMO.

Reply:
there will probably be a `.snyk` file appear at some point

Reply:
But that won't rebuild the image?